packages/block-editor/src/components/inserter/index.native.js

```diff
@@ -71,10 +55,9 @@ const defaultRenderToggle = ( {
 				onLongPress,
 			} }
 			isDisabled={ disabled }
-			{ ...expandedModeViewProps }
-		>
-			{ useExpandedMode && expandedModeViewText }
-		</ToolbarButton>
+			customContainerStyles={ buttonStyle }
+			fixedRatio={ false }
```

Comment:
By testing it a bit, I think we should add some `hitSlop` to this button, currently, this is not implemented but it should be easy to do and it will improve the UX of tapping on this button. What do you think?

Reply:
Good idea, updated.